### PR TITLE
jena 3.0.1

### PR DESCRIPTION
**Pull request by joernhees**
removed files aren't present in the recent version anymore

### CODE REVIEW

File: Library/Formula/jena.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Jena < Formula
   desc "Framework for building semantic web and linked data apps"
   homepage "https://jena.apache.org/"
-  url "https://archive.apache.org/dist/jena/binaries/apache-jena-2.12.1.tar.gz"
-  sha256 "5bbda78a873c1f27af1719e1d1e724a56a0739d90a9450fc3b4b710ad128734b"
+  url "http://archive.apache.org/dist/jena/binaries/apache-jena-3.0.1.tar.gz"
```
Comment (DomT4) [inline Library/Formula/jena.rb]: `* Stable: Please use https:// for http://archive.apache.org/dist/jena/binaries/apache-jena-3.0.1.tar.gz`

Comment (joernhees) [inline Library/Formula/jena.rb]: oups, sorry copy paste error...

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/6ef30b6485dbc749b9475c5f00c838bca34759a5. Thank you for your contribution to Homebrew @joernhees; we appreciate it! :smiley_cat:

**Comment by joernhees** [comment]
:+1: hugs for homebrew ;)

**Pull request closed without merging**